photos.html

```diff
@@ -1 +1,229 @@
+      <div class="container">
+        <!-- Brand and toggle get grouped for better mobile display -->
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="homepage.html">
+            <img src="img/danish_flag.gif" height="23px" alt="Danebook Icon" class="img-circle">
+            Danebook
+          </a>
+        </div>
+
+        <!-- Collect the nav links, forms, and other content for toggling -->
+        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+          <form class="navbar-form navbar-left" role="search">
+            <div class="form-group">
+              <input type="text" class="form-control" placeholder="Search for users" id="user-search-bar">
+            </div>
+          </form>
+          <ul class="nav navbar-nav navbar-right">
+            <li><a href="#">Harry</a></li>
+          </ul>
+        </div><!-- /.navbar-collapse -->
+      </div><!-- /.container-fluid -->
+    </nav>
+
+    <header>
+      <div class="jumbotron">
+        <div id="photos-panel" class="container">
+          <img src="img/hogwarts_small.jpg" alt="User Cover Photo" id="cover-photo">
+          <img id="profile-photo" src="img/harry_potter_small.jpg" alt="User Profile Photo" height="200px">
+          <a id="user-name">Harry Potter</a>
+          <!-- Row with page names -->
+          <ul class="nav nav-tabs">
+            <li role="presentation" class="col-md-offset-2">
+              <a href="timeline.html">Timeline</a>
+            </li>
+            <li role="presentation">
+              <a href="about.html">About</a>
+            </li>
+            <li role="presentation" class="active">
+              <a href="photos.html">Photos (123)</a>
+            </li>
+            <li role="presentation">
+              <a href="friends.html">Friends (542)</a>
+            </li>
+            <li role="presentation" class="col-md-offset-3">
+              <a href="about_edit.html">Edit Profile</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </header>
+
+
+    <main class="container" id="photos">
+      <div class="row section-banner">
+        <div class="col-md-2 col-md-offset-5 section-header">
+          <h1>Photos</h1>
+        </div>
+        <div class="col-md-2 col-md-offset-3 edit-button">
+          <button>Add photo!</button>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-md-3 captioned-photo">
+          <a href="#">
+            <img src="img/icon_photo_small.png" class="img-responsive">
+          </a>
+          <div class="caption">
```

Comment:
I noticed some issues with the captions not matching the width of the photos exactly on various window sizes. This can be fixed with a simple change to the width in the CSS, to 100% of the parent class.